docs/paper/dev/api/components.md

```diff
@@ -52,13 +52,17 @@ builders for all the types. These objects are immutable so when constructing mor
 recommended to use builders to avoid creating new Component instances with every change.
 
 ```java
-final Component component = Component.text("Hello world!", NamedTextColor.RED);
-
-// the adventure component library is built with static imports in mind to make code more consise.
+// This is a sub-optimimal construction of the component as each change creates a new component
+final Component component = Component.text("Hello")
+    .color(TextColor.color(0x13f832))
+    .append(Component.text(" world!", NamedTextColor.GREEN));
+
+// This is an optimal use of the builder to create the same component
+// Also note that adventure Components are designed for use with static method imports 
+// to make code less verbose
 final Component component = text()
```

Comment:
Should be `final Component component = Component.text()` right?

Reply:
the doc likely assumes static imports

Reply:
Yeah, I specifically mention in the comment that its demonstrating the usefulness of static imports to make code less verbose.